### ISSUE TITLE
Fullpage V3 doesn't initiate if using a class for the wrapper element

### ISSUE THREAD

**Issue by verticalgrain** [alvarotrigo/fullPage.js#3433]
### Description
Fullpage V3 doesn't initiate if the wrapper element used is a class. If using an ID, fullpage initiates as expected.

### Link to isolated reproduction with no external CSS / JS
https://codepen.io/verticalgrain/pen/MPabav

### Versions 
Chrome 69, Mac OS

**Comment by alvarotrigo** [comment]
Thanks for reporting it!

I've fixed it [on the dev branch now](https://github.com/alvarotrigo/fullPage.js/blob/dev/src/fullpage.js). To be merged in the next release 3.0.4.

However, you can also fix it by using the new vanilla JS initialization instead. Update your codepen JS code to this:

```javacript
$(document).ready(function() {
	new fullpage('.fullpage', {
		//options here
    licenseKey:'OPEN-SOURCE-GPLV3-LICENSE',
		autoScrolling:true
	});
});
```

**Comment by alvarotrigo** [comment]
Now fixed in fullPage.js 3.0.4!
